packages/paste-core/components/avatar/stories/index.stories.tsx

```diff
@@ -84,8 +89,25 @@ export const Icon = (): React.ReactNode => {
   );
 };
 
-Image.story = {
-  parameters: {chromatic: {delay: 3000}},
+export const ColorVariants = (): React.ReactNode => {
+  return (
+    <Stack orientation="horizontal" spacing="space40">
+      <Avatar color="default" name="avatar example" icon={UserIcon} />
+      <Avatar color="decorative10" name="avatar example" icon={UserIcon} />
+      <Avatar color="decorative20" name="avatar example" icon={UserIcon} />
+      <Avatar color="decorative30" name="avatar example" icon={UserIcon} />
+      <Avatar color="decorative40" name="avatar example" icon={UserIcon} />
+    </Stack>
+  );
+};
+
+export const ShapeVariants = (): React.ReactNode => {
+  return (
+    <Stack orientation="horizontal" spacing="space40">
+      <Avatar shape="user" name="avatar example" icon={UserIcon} />
+      <Avatar shape="entity" name="avatar example" icon={BusinessIcon} />
```

Comment:
suggestion: it might be a good addition to also have a User and Entity Image avatar to cover the shape when it's also an image background

Reply:
What would be an example of an entity image?

Reply:
For the purpose of VRT, any old image would be just fine. You can even just reuse one of the human images if you didn't want to source one.